RFC/src/RFC-0202_TariScriptOpcodes.md

```diff
@@ -325,6 +325,28 @@ push 0.
 Identical to [`CheckSig`](#checksigmsg), except that nothing is pushed to the stack if the signature is valid, and the
 operation fails with `VERIFY_FAILED` if the signature is invalid.
 
+##### ConvertToRistrettoPoint,
+
+Pop the top element, converts it to a Ristretto point, and pushes this to the stack.
```

Comment:
Yeah, I realized this, and for now, the script mentions Hash as it can be used as 32 bytes. Left his out for now till we can discuss what route we want forward with this.

Reply:
The correct approach in my view would be to define a new `StackItem` type `RistrettoScalar(RisrettoSecretKey)`

```suggestion
Posp the top element, which must be a valid Ristretto scalar, calculates the corresponding Ristretto point and pushes this to the stack.
```

Reply:
Ah yup that makes sense